core/objects.go

```diff
@@ -79,6 +80,8 @@ func ValidChallenge(name string) bool {
 		fallthrough
 	case ChallengeTypeTLSSNI01:
 		fallthrough
+	case ChallengeTypeTLSSNI02:
+		fallthrough
```

Comment:
This should only `fallthrough` if `features.Enabled(features.AllowTLS02Challenges) == true`.